packages/yoroi-extension/app/Routes.js

```diff
@@ -220,11 +213,6 @@ const SettingsSubpages = (stores, actions) => (
       path={ROUTES.SETTINGS.SUPPORT}
       component={(props) => <SupportSettingsPage {...props} stores={stores} actions={actions} />}
     />
-    <Route
-      exact
-      path={ROUTES.SETTINGS.LEVEL_OF_COMPLEXITY}
-      component={(props) => <ComplexityLevelSettingsPage {...props} stores={stores} actions={actions} />}
-    />
```

Comment:
@AhmedIbrahim336 , I have confirmed it when you asked if we also need to remove the settings complexity page - this is my bad, but this was incorrect. We need only to remove the initial complexity screen in the first lauch process, but leave the settings-complexity page so users can change it later, if needed.

Let's plz revert the changes on the settings page and bring it back.

Reply:
Okay